src/test/Testcafe/testRunner.js

```diff
@@ -19,12 +19,12 @@ createTestCafe()
         selectorTimeout: 2000,
         assertionTimeout: 1000,
         pageLoadTimeout: 1000,
-        speed: 0.9,
+        speed: 1,//0.75,
```

Comment:
Remove comment? Or do you need it?

Reply:
removed, comment was there to quickly change test speeds while debugging 👏